include/fmt/format.h

```diff
@@ -3504,15 +3504,15 @@ FMT_END_NAMESPACE
 #define FMT_STRING_IMPL(s, ...)                              \
   [] {                                                       \
     /* Use a macro-like name to avoid shadowing warnings. */ \
-    struct FMT_STRING : fmt::compile_string {                \
+    struct INVOKE_FMT_STRING : fmt::compile_string {         \
```

Comment:
I suggest renaming to `FMT_COMPILE_STRING` to follow the naming convention for {fmt} macros as per comment above.

Reply:
Updated to use FMT_COMPILE_STRING as suggested.